### DISCUSSION TITLE
Quickstart for kratos gives error:ERROR: manifest for oryd/kratos:v0.8.0-alpha.4.pre.0-sqlite not found: manifest unknown: manifest unknown"

### DISCUSSION THREAD

**Discussion by cugarteblair** [ory/kratos#1944] [Q&A] [answered]
When running the [kratos quickstart]( https://www.ory.sh/kratos/docs/quickstart/ )
I get the following error:
ERROR: manifest for oryd/kratos:v0.8.0-alpha.4.pre.0-sqlite not found: manifest unknown: manifest unknown
I see this quickstart was just updated today.

Is there something I am missing?

**Comment by aeneasr** [comment, marked as answer]
The release is currently still processing :) https://app.circleci.com/pipelines/github/ory/kratos/5422/workflows/a107a7bb-f743-487a-85b8-28f356502aa1

You can use tag `oryd/kratos:v0.8.0-alpha.3-sqlite` in the meanwhile!

**Reply by cugarteblair** [reply]
Thanks! that worked.
